### DISCUSSION TITLE
GrapesJS developer + consultant

### DISCUSSION THREAD

**Discussion by GoodPHP** [GrapesJS/grapesjs#5851] [Ideas]
Hello,

I'm offering my expertise in GrapesJS development. With extensive experience in this area, I am also available for team consultations. Please let me know how I can assist you.

My plugins - https://gjs.market/authors/devfuture-development

Skype: maxim2356
Email: support@devfuture.pro

**Comment by glaprida** [comment]
I have worked with him and everything went smoothly. Will probably work again in the future. Recomended dev.
